src/render/dom/utils/css-variables-conversion.ts

```diff
@@ -15,7 +15,8 @@ function isCSSVariable(value: any): value is string {
  *
  * @param current
  */
-export const cssVariableRegex = /var\((--[a-zA-Z0-9-_]+),? ?([a-zA-Z0-9 ()%#.,-]+)?\)/
+export const cssVariableRegex =
```

Comment:
Should we also cover some unit tests for the regex?

Reply:
The function that uses it, `parseCSSVariable`, is already covered by unit tests.